README.md

```diff
@@ -46,18 +46,18 @@ which makes it easy for development teams to get started.
   <dependency>
     <groupId>com.google.cloud.bigtable</groupId>
     <artifactId>bigtable-hbase-1.x</artifactId>
-    <version>2.0.0-beta2</version>
+    <version>2.0.1-beta2</version>
```

Comment:
do not merge- version numbers need to be fixed

Reply:
done